src/api/app/models/package.rb

```diff
@@ -1231,6 +1235,12 @@ def commit(rev = nil)
     commit = fetch_rev_from_history_cache(rev)
     return commit if commit
 
+    if rev&.to_i == self.rev&.to_i
+      doc = Xmlhash.parse(source_file('_history', limit: 1))
+      cache_revision_commits(doc)
```

Comment:
This call will store in cache only one element. Is that what we want?

Reply:
I see.

I hoped we could paginate through the `_history` using this endpoint of the backend API : https://github.com/openSUSE/open-build-service/blob/b637d1118d0c9b3a6937959a281fb01cd13e19df/src/backend/bs_srcserver#L7610

But it looks like it is not supported.